app/handlers/analysis.py

```diff
@@ -18,4 +26,76 @@ class AnalysisHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
+        identifier = self.request.body_arguments["identifier"]
```

Comment:
Agreed, definitely something that needs to be done before public release. Should create an error handling branch after MVP is complete.